### PR TITLE
dotenv example and small server/index.js changes

### PR DESCRIPTION
**Pull request by claire-melbourne**
added a log of port back to my server.
added dotenv (example file includes mongoose uri)
separate scripts for dev and production webpack
changed variable name for model from Sites to Site to be consistent with db files

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 const express = require('express');
-const Sites = require('../db/Site.js');
+const Site = require('../db/Site.js');
```
Comment (owenwexler) [inline server/index.js:2]: careful with changing Mongoose model names after creation, might break database access for others using your service

**Review by owenwexler — APPROVED** [review]
generally LGTM but added a minor precautionary note.

File: server/index.js (hunk 1 above)
Comment (claire-melbourne) [reply server/index.js:2]: model name is the same-- just that I had accidentally given it a different variable name when requiring it in my server. They are called Site at the mongoose level and wanted to remain consistent.

Comment (owenwexler) [reply server/index.js:2]: Ah, got it, I misread that.

**Pull request merged**